frontend/app/src/components/home/ChatMessage.svelte

```diff
@@ -696,16 +699,6 @@
                             on:remindMe={remindMe} />
                     {/if}
                 </div>
-
```

Comment:
This still needs to be present for mobile otherwise we have no way to launch the emoji picker for reactions. 

Unless we have something inside the menu on mobile to show the favourite emojis + something to launch the picker.